### PR TITLE
Willig wants to merge Collin's Config files so we can point to them thru hebi-docs

### PR DESCRIPTION
**Pull request by HEBI-Willig**
I currently have them pointing to the `develop/config` branch in my `hebi-docs` update/PR but I will update all the links to master if we can get these confirmed and tested (when you have the chance Florian)

### CODE REVIEW

**Comment by HEBI-Willig** [comment]
@ennerf our biggest question mark is double shoulder stuff and if that's done correctly.... (A-2099-07, A-2303-01, A-2583-01)

**Comment by ennerf** [comment]
1. Before merging to main, we need to update to the release stable version matlab-2.0
2. The place for common hebi-config files is https://github.com/HebiRobotics/robot-config . That is already referenced from the docs.

I probably won't have time to look at it until I come back from travels on Thursday

**Comment by HEBI-Willig** [comment]
AHHH Another repo! Ok! thanks! no worries!!! We'll get everything copied over to https://github.com/HebiRobotics/robot-config

**Comment by ennerf** [comment]
> @ennerf our biggest question mark is double shoulder stuff and if that's done correctly.... (A-2099-07, A-2303-01, A-2583-01)

I just looked at `A-2099-07` and on first glance it looks right to me

File: kits/arms/config/A-2099-07G.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# X-Series 6-DoF Arm with Gripper
+version: 1.0
+families: ["Arm"]
+names: ["J1_base", "J2_shoulder1A", "J3_shoulder2", "J4_elbow1", "J5_elbow2", "J6_wrist1", "J7_wrist2"]
+hrdf: "../hrdf/A-2099-07G.hrdf"
+
+gains:
+  default: "../gains/A-2099-01=7.xml"
```
Comment (ennerf) [inline kits/arms/config/A-2099-07G.yaml:8]: looks like `=` is a typo

**Comment by HEBI-Willig** [comment]
@colbarack will get working on the move to the other repo. We'll still keep them here though or would you like them removed?

**Pull request closed without merging**